### ISSUE TITLE
postcss module not found in latest (0.74.0) docker image

### ISSUE THREAD

**Issue by lylebrown** [kromitgmbh/titra#118]
Just grabbed the latest docker image and I'm running into the following error:

```
Starting titra...,
    at /app/bundle/programs/server/boot.js:464:7,
Error: Cannot find module "/node_modules/meteor/minifier-css/node_modules/postcss/lib/postcss.js". Try installing the npm package or make sure it is not a devDependency.,
/app/bundle/programs/server/node_modules/fibers/future.js:280,
						throw(ex);,
						^,
    at Module.useNode (packages/modules-runtime.js:647:11),
    at module (packages/minifier-css/minifier.js:297:1),
    at fileEvaluate (packages/modules-runtime.js:336:7),
    at Module.require (packages/modules-runtime.js:238:14),
    at Module.moduleLink [as link] (/app/bundle/programs/server/npm/node_modules/meteor/modules/node_modules/@meteorjs/reify/lib/runtime/index.js:52:22),
    at packages/minifier-css/minifier.js:1:230,
    at require (packages/modules-runtime.js:258:21),
    at packages/minifier-css/minifier.js:297:1,
    at /app/bundle/programs/server/boot.js:401:38,
    at Array.forEach (<anonymous>),
    at /app/bundle/programs/server/boot.js:226:21
```
Is there some kind of setup that needs to be run after an update?

**Comment by faburem** [comment]
fixed in titra 0.74.2!

**Comment by lylebrown** [comment]
Hi,

Unfortunately I don't think this is fully fixed. The frontend date selection looks good now, but it's still recording all of my new entry dates as 1970-01-01. No error messages in the application log or the browser console either.

**Comment by lylebrown** [comment]
I wanted to confirm that this was fixed in the latest version (0.74.3), but has introduced another date related issue (#123). Thank you for your continued support of this application!
